spyder/plugins/pylint/widgets/pylintgui.py

```diff
@@ -499,9 +505,7 @@ def show_data(self, justanalyzed=False):
                     text_prun = ' (%s %s/10)' % (text_prun, previous_rate)
                     text += prevrate_style % (self.prevrate_color, text_prun)
                 self.treewidget.set_results(filename, results)
-                date = to_text_string(time.strftime("%Y-%m-%d %H:%M:%S",
-                                                    datetime),
-                                      encoding='utf8')

```

Comment:
Why there's no need to convert this string to `utf8`?

Reply:
Because the time.strftime returns a string, so there's no need to cast it

Reply:
Ok, nice! :+1: